Library/Formula/qt5.rb

```diff
@@ -72,6 +74,13 @@ class Qt5 < Formula
   depends_on OracleHomeVarRequirement if build.with? "oci"
 
   def install
+    # Save users some frustration by telling them that `--cc=gcc-5` or similar
+    # won't work. It won't be picked up by the Qt build system, there is no easy
+    # way to select an arbitrary compiler, and it's not supported anyway.
+    if ENV.compiler != :clang
```

Comment:
Can you explain why this is better? To me it feels more repetitive, unlike the `raise` version it doesn't tell the user what's wrong, and it will break (or require adjustment) once GCC 6 is released.